conf/test_fasta_reads.config

```diff
@@ -20,6 +20,7 @@ params {
   // TODO nf-core: Give any required params for the test so that command line flags are not needed
   reads = 'data/test/sample*.fasta'
   ref = 'data/MN908947.3.fa'
+  ref_host = 'data/human_chr1.fa'
```

Comment:
Please add this line to `conf/test.config` as well (which is the main test file -- `test_fasta_reads.config` is outdated and will be removed in future refactor).